src/main/java/org/jabref/gui/collab/DatabaseChangeMonitor.java

```diff
@@ -43,7 +44,7 @@ public DatabaseChangeMonitor(BibDatabaseContext database,
                                  PreferencesService preferencesService,
                                  StateManager stateManager,
                                  ThemeManager themeManager,
-                                 LibraryTab.DatabaseNotification notificationPane) {
+                                 LibraryTab libraryTab) {
```

Comment:
Please revert this change - in line 66 you get the notification pane. I would assume, it's the same pane - or is that pane a different one?